### ISSUE TITLE
Make links keyboard tab-able

### ISSUE THREAD

**Issue by katevoss** [catalpainternational/canoe#250]
On the Login screen there are 2 links (Forgot username and Guest user) which I cannot use my keyboard to Tab to and click (with Enter). This should be possible for accessibility.
